src/geo/bboxUtils.ts

```diff
@@ -15,28 +15,31 @@ const snapMinCordToTileGrid = (cord: number, tileRes: number): number => {
  * @returns bbox that contains the original bbox and match tile grid lines
  */
 export const snapBBoxToTileGrid = (bbox: BBox2d, zoomLevel: number): BBox2d => {
+  const initValue = -9999;
+  const snappedBbox: BBox2d = [initValue, initValue, initValue, initValue];
```

Comment:
if you want to initialize a value use a valid BBOX of -180 -90 90 180

Reply:
done